demo/admin/src/products/future/ProductForm.cometGen.tsx

```diff
@@ -29,7 +29,7 @@ export default defineConfig<GQLProduct>({
                 },
                 { type: "text", name: "slug" },
                 { type: "date", name: "createdAt", label: "Created", readOnly: true },
-                { type: "text", name: "description", label: "Description", multiline: true },
+                { type: "text", name: "description", label: "Description", multiline: true, required: false },
```

Comment:
The description column is "required" in the database. Shouldn't we change the default behavior to store an empty string even if the field isn't marked as not required in the cometGen?

Reply:
> The description column is "required" in the database.

The description column is not nullable in the database. "required" is different, as that doesn't allow empty string - but the database does.

> Shouldn't we change the default behavior to store an empty string even if the field isn't marked as not required in the cometGen?

what do you want to change?

Reply:
I‘m thinking about always allowing saving empty strings (without explicitly setting required to false) and making text fields only required if they are explicitly set to required in the config. This would better reflect the behavior in the API.

Reply:
:+1: I thought about that too when working on this.

I'll change that in a future PR

Reply:
https://github.com/vivid-planet/comet/pull/3775